.github/actions/smoke-test/build.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -o errexit 
+set -o nounset
+
+readonly TEMPLATE_ID="$1"
+
+shopt -s dotglob
```

Comment:
Since it is non-standard, please add a comment regarding why this is enabled, and be sure it's use is minimally scoped to where it is needed; disable it when it is no longer needed.